core/src/main/java/com/novoda/noplayer/drm/LocalDrmSessionManager.java

```diff
@@ -17,14 +19,27 @@
 
 class LocalDrmSessionManager implements DrmSessionManager<FrameworkMediaCrypto> {
 
-    private static final UUID WIDEVINE_MODULAR_UUID = new UUID(0xEDEF8BA979D64ACEL, 0xA3C827DCD51D21EDL);
-
     private final KeySetId keySetIdToRestore;
     private final ExoMediaDrm<FrameworkMediaCrypto> mediaDrm;
+    private final DefaultDrmSessionManager.EventListener eventListener;
+    private final UUID drmScheme;
+    private final Handler handler;
+
+    static LocalDrmSessionManager newInstance(KeySetId keySetIdToRestore, ExoMediaDrm<FrameworkMediaCrypto> mediaDrm, DefaultDrmSessionManager.EventListener eventListener, UUID drmScheme) {
```

Comment:
I think the `UUID` parameter would feel better next to the `ExoMediaDrm` and leaving the listener for the last. Same for the constructor. 

Of course this is not a blocker